test/test_model_roi.py

```diff
@@ -94,4 +94,3 @@ def test_create_roi():
     )
     assert (first_contour.ContourGeometricType == "CLOSED_PLANAR")
     assert (rt_ss.RTROIObservationsSequence[0].RTROIInterpretedType == "ORGAN")
-    
```

Comment:
please add unit test for generate_rtss_from_ct function

Reply:
any ideas how to test this function ?